static/src/js/util/accessMethods/__tests__/defaultConcatenation.test.js

```diff
@@ -0,0 +1,50 @@
+import { defaultConcatenation } from '../defaultConcatenation'
+
+describe('defaultConcatenation', () => {
+  describe('when concatenation is checked by default', () => {
+    test('returns true', () => {
+      const response = defaultConcatenation({
+        conceptId: 'S100000-EDSC',
+        serviceOptions: {
+          aggregation: {
+            concatenate: {
+              concatenateDefault: true
+            }
+          }
+        }
+      })
+
+      expect(response).toBeTruthy()
+    })
+  })
+
+  describe('when concatenation is not in aggregation', () => {
+    test('returns false', () => {
+      const response = defaultConcatenation({
+        conceptId: 'S100000-EDSC',
+        serviceOptions: {
+          aggregation: {}
+        }
+      })
+
+      expect(response).toBeFalsy()
+    })
+  })
+
+  describe('when concatenation is not supported', () => {
```

Comment:
nit; isn't this the assertion that we want? Since even if concatenationDefault is false we are still "supporting" concatenation?
```suggestion
  describe('when concatenationDefault is set to false', () => {
```